packages/app-js/src/snippets/storage-examples.ts

```diff
@@ -11,12 +11,13 @@ export const storageGetInfo: Snippet = {
   code: `// Get chain state information
 // Make our basic chain state/storage queries, all in one go
 
-const [minimumPeriod, validators, transferFee] = await Promise.all([
+const [minimumPeriod, validators] = await Promise.all([
```

Comment:
minimumPeriod is also gone :(

Reply:
I replaced it wit `blockPeriod ` in the example